test/Makefile

```diff
@@ -0,0 +1,15 @@
+CC = gcc
```

Comment:
This need to be a CMakeLists.txt

Reply:
If we use CMakeLists.txt, do we need to include it in our main CMakeLists.txt?

Reply:
The test is an individual feature. If we include it in our main `CMakeLists.txt`, it will affect the installation.

Reply:
Yeah, in order to get the file called.

You can exclude the test binary from the installation - it can be local to the development environment only. It could be as simple as not having something like https://github.com/pgmoneta/pgmoneta/blob/main/src/CMakeLists.txt#L266